.github/workflows/hlint.yml

```diff
@@ -0,0 +1,24 @@
+name: lint
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'releases/*'
```

Comment:
Let's move this job inside the `ci.yml` file. It already has proper `on` criteria:

https://github.com/chshersh/iris/blob/1bede5f2c7211e633c6356f03b1dc5f201326717/.github/workflows/ci.yml#L3-L10